pkg/cloudprovider/drift.go

```diff
@@ -126,20 +126,18 @@ func (c *CloudProvider) isImageVersionDrifted(
 	return "", nil
 }
 
-// TODO: remove nolint on unparam. Added for now in order to pass "make verify"
```

Comment:
We only ever return "", and never return a drift reason. So i just removed this bit to simplify the code